.github/actions/deploy/action.yml

```diff
@@ -0,0 +1,132 @@
+name: deploy
+description: deploys application
+
+inputs:
+  actions-api-access-token:
+    required: true
+  azure_credentials:
+    required: true
+  environment:
+    description: Environment to deploy to
+    required: true
+  pr:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: true
+  terraform-state-access-key:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Start ${{ inputs.environment }} Deployment
+      uses: bobheadxi/deployments@v0.6.2
```

Comment:
The current version is 1.1.0. Can we specify `@v1` instead?

Reply:
bumped to v1